.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Rust",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "args": { "VARIANT": "buster" }
```

Comment:
Is this line necessary?  It seems like it is already set via an `ARG` instruction in the `Dockerfile`.